src/app/applications/splash-modal/splash-modal.component.html

```diff
@@ -6,8 +6,8 @@
 
   <h2>Find Crown Land Applications</h2>
   <div class="intro-text">
-    <p>The Province is responsible for managing Crown land in British Columbia.  In an average year, between 5,000 to 7,000 applications for the use of Crown land are received and processed by the Province.&nbsp;&nbsp;<a routerLink="/about">Learn more</a></p>
-    <p>Use this site to find, review, and comment on active Crown Land applications across the province.</p>
+    <p>The Province is responsible for managing Crown land in British Columbia. In an average year the Province receives and processes between 5,000 and 7,000 applications for the use of Crown land. <a href="/about">Learn more about the process</a>.</p>
```

Comment:
Using 'routerLink' caused Angular to go to the About page right away.
Using 'href' causes a full reload of the web application, which is undesirable.
Is there a reason for this change? If not, please change it back. Thx.